easybuild/scripts/bootstrap_eb.py

```diff
@@ -519,6 +532,8 @@ def main():
 # EasyBuild is a (set of) Python packages, so it depends on Python
 # usually, we want to use the system Python, so no actual Python dependency is listed
 allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+preinstallopts = '%(preinstallopts)s'
```

Comment:
We need to include a (template) easyconfig for EasyBuild in the bootstrap script because:
(i) we (now) also support *offline* bootstrapping (by providing tarballs for each of the EasyBuild components)
(ii) we take control over the `easy_install`/`setuptools`/`distribute` being used by the bootstrap, to have more control

We currently don't do (ii) when installing EasyBuild with EasyBuild outside of the bootstrap, since things are less tricky there. Maybe we should look into that too in that situation, but that shouldn't block this PR, imho.

For other Python packages being installed using EasyBuild, you're usually using a Python module installed with EasyBuild, which includes a setuptools, so there it's typically not an issue.